### ISSUE TITLE
Expected ENDFORM (structure) caused by inline data / Assign

### ISSUE THREAD

**Issue by ed-holland** [abaplint/abaplint#1113]
Source inline data
![image](https://user-images.githubusercontent.com/53995991/89035474-4e651500-d33b-11ea-8201-374efb461aab.png)

Source Assign
![image](https://user-images.githubusercontent.com/53995991/89035597-83716780-d33b-11ea-98a3-f0eebee45a43.png)

**Comment by larshp** [comment]
hmm, this looks strange, the version should be one of

```
  v740sp02 = "v740sp02",
  v740sp05 = "v740sp05",
  v740sp08 = "v740sp08",
```

should give an error while editing, something like:

![image](https://user-images.githubusercontent.com/5888506/89036067-507ba380-d33c-11ea-96f0-69982ed039b8.png)

**Comment by ed-holland** [comment]
Adjusted:
![image](https://user-images.githubusercontent.com/53995991/89039832-02b66980-d343-11ea-8c19-c89e01b558ee.png)

**Comment by larshp** [comment]
try again, this was fixed a few days ago

**Comment by ed-holland** [comment]
issue resolved. Thx
